backend/backend/backend/settings.py

```diff
@@ -0,0 +1,127 @@
+"""
+Django settings for backend project.
+
+Generated by 'django-admin startproject' using Django 4.2.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-i@u3*c-lsnqi)b%8tzyfyu1k!lg0at1_!gc&^xa^%*=s8jf1nd'
```

Comment:
Do we want to commit this?

Reply:
Technically no, but we won't use it in production.  This is more of an example of how to set up docker and compose.  Good catch though.